lib/config/formatter.go

```diff
@@ -342,7 +342,7 @@ func formatCallerWithPathAndLine() (path string) {
 	return ""
 }
 
-var frameIgnorePattern = regexp.MustCompile(`github\.com/(gravitational|(S|s)irupsen)/logrus`)
+var frameIgnorePattern = regexp.MustCompile(`github\.com/((S|s)irupsen)/logrus`)
```

Comment:
Maybe we could drop the weird `(S|s)` and do a case-insensitive compare instead?

Reply:
I don't think that case-insensitive compare is a good fit as it's slower that the current one. But I also think we can drop `S` at all. Currently the logrus package is called "github.com/sirupsen/logrus" and I don't think that anyone uses the old one anymore (I checked our current dependencies and I don't see it).